language/ja/roles.yaml

```
@@ -910,6 +911,10 @@ Streamer:
 QueenOfNight:
   FascinatePlayer: '{{name}}により{{target}}の心は搔き乱されました。'
 
+Tarzan:
+  # Result of skill.
+  result: '{{name}}は{{count}}人の人狼の気配を感じた。'
```

ログは敬体で統一しているので、「感じました」に変更をお願いします。

```suggestion
  result: '{{name}}は{{count}}人の人狼の気配を感じました。'
```